ansible/roles/compute_init/tasks/main.yml

```diff
@@ -0,0 +1,133 @@
+---
+
+- name: Ensure directories exist
+  file: 
+    path: "/etc/ansible-init/{{ item.directory }}"
+    state: directory
+    owner: root
+    group: root
+    mode: 0755
+  loop:
+    - { directory: "templates" }
+    - { directory: "files" }
+    - { directory: "library" }
+    - { directory: "filter_plugins" }
+
+- name: Inject templates
+  copy:
+    src: '{{ item }}'
+    dest: '/etc/ansible-init/templates/{{ item | basename }}'
+    owner: root
+    group: root
+    mode: 0644
+  loop:
+    - ../../resolv_conf/templates/resolv.conf.j2
+    - ../../stackhpc.os-manila-mount/templates/ceph.conf.j2
+    - ../../stackhpc.os-manila-mount/templates/ceph.keyring.j2
+
+- name: Inject files
+  copy:
+    src: '{{ item }}'
+    dest: '/etc/ansible-init/files/{{ item | basename }}'
+    owner: root
+    group: root
+    mode: 0644
+  loop:
+    - ../../resolv_conf/files/NetworkManager-dns-none.conf
+
+- name: Inject files
```

Comment:
Seeing this does make me wonder if we could just inject files from e.g. `roles/myrole/tasks/runtime.yml` ...

Reply:
Why leave compute init functionality inside the roles though? I think this way is more modular and inline with the purpose of inventory groups. Plus keeping everything centralised aids readability for compute init. (If I'm understanding you)